src/MLHeader/style/index.js

```diff
@@ -0,0 +1,2 @@
+import 'antd/es/upload/style'
```

Comment:
Careful with your copy-pastes

Reply:
Gah. Hopefully we'll get permission to make a template.